kernel_config

```diff
@@ -791,26 +791,26 @@ CONFIG_INET_UDP_DIAG=m
 CONFIG_TCP_CONG_CUBIC=y
 CONFIG_DEFAULT_TCP_CONG="cubic"
 # CONFIG_TCP_MD5SIG is not set
-CONFIG_IPV6=m
+CONFIG_IPV6=y
 CONFIG_IPV6_ROUTER_PREF=y
 # CONFIG_IPV6_ROUTE_INFO is not set
 # CONFIG_IPV6_OPTIMISTIC_DAD is not set
 CONFIG_INET6_AH=m
-CONFIG_INET6_ESP=m
+CONFIG_INET6_ESP=y
```

Comment:
Why is this being changed from `=m`?  Debian uses `=m` here, so I'd rather stick to `=m` everywhere we can (especially when Debian's kernel does).